lib/faker/default/slack_emoji.rb

```diff
@@ -3,38 +3,119 @@
 module Faker
   class SlackEmoji < Base
     class << self
+      ##
+      # Produces a random slack emoji from people category
```

Comment:
Can you add a period at the end of the sentence?

```suggestion
      # Produces a random slack emoji from people category.
```

The same applies to the other sentences in this PR.